### PR TITLE
Add current-level quality declarations

### PR DESCRIPTION
**Pull request by brawner**
Adding Quality Declarations for each package with the current QL of 4.

Depends on #108 
New commit: b3f47837b1aceda17e437bb0187ca65e18646a9d

### CODE REVIEW

File: sensor_msgs/QUALITY_DECLARATION.md [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+
+All message and service definition files located in `include`, `msg` and `srv` directories are considered part of the public API.
+
+### API Stability Within a Released ROS Distribution [1.iv]/[1.vi]
+
+`sensor_msgs` will not break public API within a released ROS distribution, i.e. no major releases once the ROS distribution is released.
+
+### ABI Stability Within a Released ROS Distribution [1.v]/[1.vi]
+
+`sensor_msgs` contains c++ code and is therefore concerned with ABI stability. It will not break ABI stability within a released ROS distribution.
```
Comment (ahcorde) [inline sensor_msgs/QUALITY_DECLARATION.md:29]:
```suggestion
`sensor_msgs` contains C++ code and is therefore concerned with ABI stability. It will not break ABI stability within a released ROS distribution.
```

File: sensor_msgs/QUALITY_DECLARATION.md [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+`sensor_msgs` is a package providing strictly message and service definitions and therefore does not require associated tests and has no coverage or performance requirements.
+
+### Public API Testing [4.ii]
+
+Each part of the public non-generated c++ API has tests, and new additions or changes to the public API require tests before being added.
+The tests aim to cover both typical usage and corner cases, but are quantified by contributing to code coverage.
+
+### Coverage [4.iii]
+
+`rcpputils` does not currently track test coverage.
```
Comment (ahcorde) [inline sensor_msgs/QUALITY_DECLARATION.md:89]: package name and below

File: sensor_msgs/QUALITY_DECLARATION.md [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+
+Most recent test results can be found [here](http://build.ros2.org/view/Epr/job/Epr__common_interfaces__ubuntu_bionic_amd64/lastBuild/testReport/sensor_msgs/copyright/)
+
+### Copyright Statements [3.iv]
+
+There are no currently copyrighted source files in this package.
+
+## Testing [4]
+
+`sensor_msgs` is a package providing strictly message and service definitions and therefore does not require associated tests and has no coverage or performance requirements.
```
Comment (ahcorde) [inline sensor_msgs/QUALITY_DECLARATION.md:80]: some headers ? do we need to test them?

File: sensor_msgs/QUALITY_DECLARATION.md (hunk 1 above)
Comment (brawner) [reply sensor_msgs/QUALITY_DECLARATION.md]: Addressed in all

File: sensor_msgs/QUALITY_DECLARATION.md (hunk 2 above)
Comment (brawner) [reply sensor_msgs/QUALITY_DECLARATION.md]: Fixed.

File: sensor_msgs/QUALITY_DECLARATION.md (hunk 3 above)
Comment (brawner) [reply sensor_msgs/QUALITY_DECLARATION.md]: Yes, most of them are actually being tested. Updating this section to reflect that.

File: sensor_msgs/QUALITY_DECLARATION.md [hunk 4]
```diff
@@ -0,0 +1,132 @@
...
+### Copyright Statements [3.iv]
+
+There are no currently copyrighted source files in this package.
+
+## Testing [4]
+
+### Feature Testing [4.i]
+
+Most of the features in sensor_msgs have corresponding tests which simulate typical usage, and they are located in the `test` directory.
+New features are required to have tests before being added.
```
Comment (ahcorde) [inline sensor_msgs/QUALITY_DECLARATION.md:83]: I checked that there is a test for the code in the [CMakeLists.txt](https://github.com/ros2/common_interfaces/blob/master/sensor_msgs/CMakeLists.txt#L68). But some some reason I can't see the test in the [buildfarm](http://build.ros2.org/view/Epr/job/Epr__common_interfaces__ubuntu_bionic_amd64/lastBuild/testReport/sensor_msgs/

Comment (brawner) [reply sensor_msgs/QUALITY_DECLARATION.md:83]: Weird, it's not under sensor_msgs, it's under (root). Added a link.
